Mathlib/MeasureTheory/Function/LpSeminorm/Basic.lean

```diff
@@ -985,6 +1053,10 @@ theorem eLpNorm'_eq_zero_iff (hq0_lt : 0 < q) {f : α → E} (hf : AEStronglyMea
     eLpNorm' f q μ = 0 ↔ f =ᵐ[μ] 0 :=
   ⟨ae_eq_zero_of_eLpNorm'_eq_zero (le_of_lt hq0_lt) hf, eLpNorm'_eq_zero_of_ae_zero hq0_lt⟩
 
+theorem enorm_ae_le_eLpNormEssSup {_ : MeasurableSpace α} (f : α → ε) (μ : Measure α) :
+    ∀ᵐ x ∂μ, ‖f x‖ₑ ≤ eLpNormEssSup f μ :=
```

Comment:
Has the lemma below changed since the start of this PR? Now it states the same, and should be renamed (+ deprecated) to this name.

Reply:
I think it did, perhaps Yael's PR. Good catch!, fixed.